tests/integration_tests/instances.py

```diff
@@ -186,6 +186,7 @@ def install_deb(self):
             local_path=integration_settings.CLOUD_INIT_SOURCE,
             remote_path=remote_path,
         )
+        assert self.execute("apt-get install -qy python3-passlib").ok
```

Comment:
This isn't the right place to put this. It should probably go in integration_tests/conftest.py/setup_image or perhaps the test itself.

Reply:
I updated this to be an integration test requirement.